Failure Building binutils for Bare Metal AArch64

Here is the message I am getting when I am running `ct-ng build` for a 64 bit arm target (AArch64) with the OS selected as bare-metal:

```
[ALL  ]    *** BFD does not support target aarch64-unknown-eabi.
[ALL  ]    *** Look in bfd/config.bfd for supported targets.
[ALL  ]    Makefile:2676: recipe for target 'configure-bfd' failed
[ERROR]    make[3]: *** [configure-bfd] Error 1
[ALL  ]    make[3]: Leaving directory '/home/steven/src/cross/.build/aarch64-unknown-eabi/build/build-binutils-host-x86_64-build_pc-linux-gnu'
[ALL  ]    Makefile:849: recipe for target 'all' failed
[ERROR]    make[2]: *** [all] Error 2
[ALL  ]    make[2]: Leaving directory '/home/steven/src/cross/.build/aarch64-unknown-eabi/build/build-binutils-host-x86_64-build_pc-linux-gnu'
[ERROR]  
[ERROR]  >>
[ERROR]  >>  Build failed in step 'Installing binutils for host'
[ERROR]  >>        called in step '(top-level)'
```

crosstool-NG uses a tuple of `aarch64-unknown-eabi` with this configuration. I get the same error when I manually build binutils configured with `--target=aarch64-unknown-eabi`. It appears from looking in `bfd/config.bfd`(as the error message suggests) that the appropriate tuple should be `aarch64-unknown-elf`. It does work when I manually build binutils configured with `--target=aarch64-unknown-elf`. Linaro uses a similar tuple when describing the aarch64, bare-metal tools on their download page: see [downloads](https://www.linaro.org/downloads/) at the bottom of the page.

### Support Details
crosstool-NG version: crosstool-ng-1.23.0
host OS and its version: Ubuntu 17.04
.config: [.config](https://gist.github.com/sbosnick/15167655e7213474724eb5462623e032)
build.log: [build.log](https://gist.github.com/sbosnick/f41afa39dd4aa909dcef6d8609814c67)

--- a/config/arch/arm.in.2
+++ b/config/arch/arm.in.2
@@ -1,5 +1,6 @@
 # ARM specific configuration file
 
+if ARCH_32
 config ARCH_ARM_MODE
     string
     default "arm"   if ARCH_ARM_MODE_ARM
@@ -91,3 +92,5 @@ config ARCH_ARM_TUPLE_USE_EABIHF
 
       Say 'n', unless you are trying to fix gcc to properly recognise
       the *eabihf tuples.
+
+endif

--- a/config/target.in
+++ b/config/target.in
@@ -144,6 +144,9 @@ config ARCH_SUPPORTS_64
 config ARCH_DEFAULT_8
     bool
 
+config ARCH_DEFAULT_16
+    bool
+
 config ARCH_DEFAULT_32
     bool
 
@@ -153,6 +156,7 @@ config ARCH_DEFAULT_64
 config ARCH_BITNESS
     int
     default "8"     if ARCH_8
+    default "16"    if ARCH_16
     default "32"    if ARCH_32
     default "64"    if ARCH_64
 
@@ -160,6 +164,7 @@ choice
     bool
     prompt "Bitness:"
     default ARCH_8  if ARCH_DEFAULT_8
+    default ARCH_16 if ARCH_DEFAULT_16
     default ARCH_32 if ARCH_DEFAULT_32
     default ARCH_64 if ARCH_DEFAULT_64
 

